lib/.gemspec.in

```diff
@@ -0,0 +1,20 @@
+# -*- encoding: utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = "facter"
+  version = "@LIBFACTER_VERSION_MAJOR@.@LIBFACTER_VERSION_MINOR@.@LIBFACTER_VERSION_PATCH@"
+  mdata = version.match(/(\d+\.\d+\.\d+)/)
+  s.version = mdata ? mdata[1] : version
+
+  s.required_rubygems_version = Gem::Requirement.new(">= 0")
+  s.authors = ["Puppet Labs"]
+  s.date = "2012-08-08"
```

Comment:
Should we have a more updated date? (nonblocking)

Can we specify a min ruby version above 2?